substrate/frame/benchmarking/src/v1.rs

```diff
@@ -1008,7 +1008,8 @@ macro_rules! impl_benchmark {
 					$(
 						(stringify!($pov_name).as_bytes().to_vec(),
 						$crate::__private::vec![
-							$( ( stringify!($storage).as_bytes().to_vec(),
+							// Stringify sometimes includes spaces, depending on the Rust version.
+							$( ( stringify!($storage).replace(" ", "").as_bytes().to_vec(),
```

Comment:
We should also fix it on the side where we handle these `pov_modes`. Spaces are probably never an acceptable identifier.

Reply:
added replaces